### ISSUE TITLE
Non existing message leads to error instead of intlify warning

### ISSUE THREAD

**Issue by flatcat12** [intlify/vue-i18n-next#1107]
### Reporting a bug?

Until vue-i18n 9.1.10, a missing message in the message catalog leads to a waring. 
With upgrade to vue-i18n 9.2.0, the same code crashes and cannot be executed anymore in the browser (Chrome, FF, Safari).
The error I get is:
vue-i18n.esm-bundler.js:1665 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'NODE_ENV')
    at process (vue-i18n.esm-bundler.js:1665:21)
    at register (vue-i18n.esm-bundler.js:1674:37)
    at callWithErrorHandling (vue.global.js?1655737501:1733:24)
    at callWithAsyncErrorHandling (vue.global.js?1655737501:1742:23)
    at invokeDirectiveHook (vue.global.js?1655737501:4330:15)
    at mountElement (vue.global.js?1655737501:6625:19)
    at processElement (vue.global.js?1655737501:6604:15)
    at patch (vue.global.js?1655737501:6524:23)
    at mountChildren (vue.global.js?1655737501:6712:15)
    at mountElement (vue.global.js?1655737501:6622:19)

When falling back to 9.1.10 version of the package, everything works as expected.

### Expected behavior

Non existing catalog message should yield a warning as before (that can optionally be suppressed with config settings missingWarn and fallbackWarn).

### Reproduction

Use a vue-i18n enabled project. In the template section render a message like so:

<template>
  <div v-t="'non.existing.messagekey'"></div>
</template>

### System Info

```shell
Environment with vue-i18n 9.2.0
$ npx envinfo --system --npmPackages '{vue*,@vue/*,vue-i18n*,@intlify/*,vite*,@vitejs/*}' --binaries --browsers

  System:
    OS: macOS 12.5
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 2.27 GB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.16.0 - /usr/local/bin/node
    npm: 8.11.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 104.0.5112.79
    Firefox: 103.0
    Safari: 15.6
  npmPackages:
    @vue/cli-plugin-babel: ~5.0.0 => 5.0.8 
    @vue/cli-plugin-e2e-nightwatch: ~5.0.0 => 5.0.8 
    @vue/cli-plugin-eslint: ~5.0.0 => 5.0.8 
    @vue/cli-plugin-unit-jest: ~5.0.0 => 5.0.8 
    @vue/cli-service: ~5.0.0 => 5.0.8 
    @vue/test-utils: ^2.0.0-0 => 2.0.2 
    @vue/vue3-jest: ^27.0.0-alpha.1 => 27.0.0 
    vue: ^3.2.13 => 3.2.37 
    vue-i18n: ^9.1.9 => 9.2.0 

Environment with cue-i18n 9.1.10
$ npx envinfo --system --npmPackages '{vue*,@vue/*,vue-i18n*,@intlify/*,vite*,@vitejs/*}' --binaries --browsers

  System:
    OS: macOS 12.5
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 963.12 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.16.0 - /usr/local/bin/node
    npm: 8.11.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 104.0.5112.79
    Firefox: 103.0
    Safari: 15.6
  npmPackages:
    @vue/cli-plugin-babel: ~5.0.0 => 5.0.8 
    @vue/cli-plugin-e2e-nightwatch: ~5.0.0 => 5.0.8 
    @vue/cli-plugin-eslint: ~5.0.0 => 5.0.8 
    @vue/cli-plugin-unit-jest: ~5.0.0 => 5.0.8 
    @vue/cli-service: ~5.0.0 => 5.0.8 
    @vue/test-utils: ^2.0.0-0 => 2.0.2 
    @vue/vue3-jest: ^27.0.0-alpha.1 => 27.0.0 
    vue: ^3.2.13 => 3.2.37 
    vue-i18n: 9.1.10 => 9.1.10
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/intlify/vue-i18n-next/blob/master/.github/CONTRIBUTING.md)
- [X] Read the [Documentation](https://vue-i18n.intlify.dev/)
- [X] Check that there isn't [already an issue](https://github.com/intlify/vue-i18n-next/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/intlify/vue-i18n-next/discussions)
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by kazupon** [comment]
Duplicate #1096 

I've released v9.2.2!
please check it!

Thanks!

**Comment by flatcat12** [comment]
Working as expected. Thanks!
